pkgs/development/python-modules/promnesia/default.nix

```diff
@@ -0,0 +1,69 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, beautifulsoup4
+, cachew
+, fastapi
+, HPI
+, httptools
+, logzero
+, lxml
+, mistletoe
+, more-itertools
+, mypy
+, orgparse
+, pytz
+, setuptools
+, setuptools-scm
+, sqlcipher3
+, tzlocal
+, urlextract
+, uvicorn
+, uvloop
+, watchfiles
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "promnesia";
```

Comment:
Is this a library or an application? We only keep libraries in `python-modules`. Application need to be moved elsewhere.

Reply:
It's both as far as I can tell - not sure how that changes the rules (if at all).. There is the command `promnesia`, but the configuration uses it as a library: https://github.com/karlicoss/promnesia/blob/master/src/promnesia/misc/config_example.py

Maybe it's both?